packages/webapp/src/components/Help/index.jsx

```diff
@@ -43,110 +60,124 @@ export default function PureHelpRequestPage({ onSubmit, goBack, email, phone_num
     const contactInformation = contactMethodSelection === 'email' ? email : phone_number;
     setValue(CONTACT_INFO, contactInformation);
   }, [contactMethodSelection]);
-  const submit = (data) => {
+
+  const confirmedSubmit = (data) => {
     data.support_type = data.support_type.value;
     data[data[CONTACT_METHOD]] = data.contactInfo;
     data.attachments = {};
+    data.farm_id = farm_id;
```

Comment:
I noticed in the database that there is a column for this, but it wasn't getting sent. I thought it could be helpful. Also, my original approach was going to delete the farm as part of the create ticket operation on the back end and this is left over from that